test/tools/yulopti.cpp

```diff
@@ -284,6 +431,11 @@ int main(int argc, char** argv)
 				po::value<string>(),
 				"steps to execute non-interactively"
 			)
+			(
+				"object",
+				po::value<string>(),
+				"path to a yul object in the input"
```

Comment:
```suggestion
				po::value<string>()->value_name("path"),
				"Dotted path to a specific Yul object in the source. "
				"If not specified, the top-level object is used. "
				"Only valid if the source actually contains objects (rather than a block of Yul code)."
```